core/controllers/creator_dashboard_test.py

```diff
@@ -118,12 +118,12 @@ def _run_user_stats_aggregator_job(self):
          start_computation())
         self.assertEqual(
             self.count_jobs_in_taskqueue(
-                queue_name=taskqueue_services.QUEUE_NAME_DEFAULT),
+                queue_name=taskqueue_services.QUEUE_NAME_CONTINUOUS_JOBS),
```

Comment:
Can we remove the default arg from queue_name in self.count_jobs_in_taskqueue, and thereby not specify "queue_name" here?